src/AutoRest.CSharp/Common/Output/Models/Types/ModelTypeProvider.cs

```diff
@@ -120,9 +121,30 @@ private static CSharpType GetDefaultPropertyType(in InputModelTypeUsage modelUsa
                 valueType = TypeFactory.GetOutputType(valueType);
             }
 
+            if (valueType.IsValueType && !property.IsRequired)
+            {
+                valueType = valueType.WithNullable(true);
+            }
+
             return valueType;
         }
 
+        private static FormattableString? GetPropertyDefaultValue(CSharpType propertyType, bool isRequired)
+        {
+            if (TypeFactory.IsList(propertyType))
+            {
+                if (TypeFactory.IsReadOnlyList(propertyType))
+                {
+                    return $"Array.Empty<{propertyType.Arguments[0]}>();";
+                }
+                if (!isRequired)
+                {
+                    return $"{Constant.NewInstanceOf(TypeFactory.GetPropertyImplementationType(propertyType)).GetConstantFormattable()};";
```

Comment:
`GetConstantFormattable()` returns `FormattableString`, so there is no need to wrap it into another formattable:
```cs
return Constant.NewInstanceOf(TypeFactory.GetPropertyImplementationType(propertyType)).GetConstantFormattable();
```